js/scale.js

```diff
@@ -0,0 +1,39 @@
+const buttonSmaller = document.querySelector('.scale__control--smaller');
+const buttonBigger = document.querySelector('.scale__control--bigger');
+const scaleInputElement = document.querySelector('.scale__control--value');
+const imagePreview = document.querySelector('.img-upload__preview img');
+
+const MIN_SCALE = 25;
+const MAX_SCALE = 100;
+const DEFAULT_SCALE = 100;
+const STEP = 25;
+
+const scaleImage = (value) => {
+  imagePreview.style.transform = `scale(${value / 100})`;
+  scaleInputElement.value = `${value}%`;
+};
+
+const onButtonSmallerClick = () => {
+  const currentValue = parseInt(scaleInputElement.value, 10);
+  let newValue = currentValue - STEP;
+  if (newValue < MIN_SCALE) {
+    newValue = MIN_SCALE;
+  }
+  scaleImage(newValue);
+};
```

Comment:
```suggestion
const onButtonSmallerClick = () => {
  const currentValue = parseInt(scaleInputElement.value, 10);
  let newValue = currentValue - STEP;
  
  if (newValue < MIN_SCALE) {
    newValue = MIN_SCALE;
  }
  
  scaleImage(newValue);
};
```

также магическое число 10

такая же проблема и в коллбеке далее

а вообще, ты можешь это в одну функцию объеденить, добавить атрибут value для кнопки и положить в нее increace или decrease, а потом по event.target.value получить инфомрацию, в какую кнопку был клик. И сделать масштабирование.